runtime/verbose/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ add_library(j9vrb SHARED
 	${CMAKE_CURRENT_BINARY_DIR}/ut_j9vrb.c
 )
 
-if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
+
+# Note: OMR_TOOLCONFIG=gnu also captures gnu-like compilers, such as clang
+if(OMR_TOOLCONFIG STREQUAL "gnu")
```

Comment:
UMA code only includes this for `spec.linux_x86.*`
https://github.com/eclipse/openj9/blob/f0a0cce37747b6637afda2ef59a940902e688125/runtime/verbose/module.xml#L103-L105

Reply:
Re-ran the build on master and it turns out the code now compiles without this flag. Looking into it more, but I believe that we are expecting gnu inline semantics, but with `-O3` the relevant functions get inlined and we sidestep the issue

Reply:
> but with -O3 the relevant functions get inlined and we sidestep the issue

You are saying a debug build would fail without this change?

Reply:
yes, I believe that is the case